conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -64,7 +80,7 @@ export CONDA_LIBMAMBA_SOLVER_NO_CHANNELS_FROM_INSTALLED=1
 {{ conda_install_tool }} update --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
     pip {{ conda_install_tool_deps }} {{ conda_build_tool_deps }} {{ " ".join(remote_ci_setup_update) }}
 {%- endif %}
-{% if local_ci_setup %}
+{% if conda_install_tool != "pixi" and local_ci_setup %}
```

Comment:
Having local ci setup configuration/code split over multiple files on a single platform is not ideal. Is there a way we can leave this here?

Reply:
Ok, I refactored this bit now, so we inject `conda` and `pip` for `local_ci_setup` cases (which is a rare exception anyway). Had to create a couple helpers to avoid duplication but it looks ok. Let me test it.